net-im/spectrum2/metadata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>ck+gentoo@bl4ckb0x.de</email>
+		<name>Conrad Kostecki</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
+		Spectrum is an open source instant messaging transport.
+		It allows users to chat together even when they are using
+		different IM networks. It acts as a transport layer between
+		the users as showed in the animation above.
+	</longdescription>
+	<use>
+		<flag name="frotz">Enable the Z-Engine backend.</flag>
+		<flag name="irc">Enable the IRC backend, based on <pkg>net-im/libcommuni</pkg>.</flag>
+		<flag name="purple">Enable the libpurple backend, based on <pkg>net-im/pidgin</pkg>.</flag>
+		<flag name="sms">Enable the SMS backend, based on <pkg>app-mobilephone/smstools</pkg>.</flag>
+		<flag name="twitter">Enable the Twitter backend, based on <pkg>net-im/swiften</pkg>.</flag>
```

Comment:
this is not correct, twitter is "based" on net-misc/curl

Reply:
Whoops. Thanks! Fixed.